org.tribler.Tribler.yaml

```diff
@@ -0,0 +1,64 @@
+id: org.tribler.Tribler
+command: tribler
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-download
+  - --persist=.Tribler
+  - --own-name=org.kde.*
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: tribler
+    buildsystem: simple
+    build-commands:
+      - ar x tribler.deb
+      - tar xf data.tar.xz
+      - mkdir -p /app/bin
+      - install tribler /app/bin
+      - install -Dm644 usr/share/applications/tribler.desktop /app/share/applications/org.tribler.Tribler.desktop
+      - |
+        for size in 64 128 256 512; do
+          install -Dm644 tribler_${size}.png /app/share/icons/hicolor/${size}x${size}/apps/org.tribler.Tribler.png || exit 1
+        done
+      - mv usr/share/tribler /app/share
+      - install -Dm644 README.md -t /app/share/tribler
+      - install -Dm644 org.tribler.Tribler.metainfo.xml -t /app/share/metainfo
+    sources:
+      - type: file
+        url: https://github.com/Tribler/tribler/releases/download/v7.8.0/tribler_7.8.0_all.deb
```

Comment:
```
[📦 org.tribler.Tribler ~]$ file /app/share/tribler/tribler
/app/share/tribler/tribler: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=1da3a1d77c7109ce6444919f4a15e7e6c63d02fa, stripped
```

This is definitely x86_64-only, just the deb is built in a wrong way. If https://github.com/Tribler/tribler/blob/main/doc/development/development_on_linux.rst is up to date, it should be easy to build it from source with https://github.com/flatpak/flatpak-builder-tools/blob/master/pip/flatpak-pip-generator

Reply:
Thought so, I'll limit it to x86 again then. 
I tried building it from source, but this being my first flatpak I had to give up after some hours of trial and error. Maybe I'll give it another try later but for now I don't have the resources unfortunately.

Reply:
it's really better to build from source.

Reply:
Can you elaborate a bit further? Why is it so much better? Especially keeping all dependencies up to date seems to me an unnecessary source of complication, when I can use a deb which is maintained by the tribler team.

Reply:
To be able to use in on `aarch64`.

Reply:
Is there an easy way to generate a manifest from apt packages? Like it works for python packages via the flatpak-pip-generator?

Reply:
short: no

longer: I don't think .deb have necessary metadata if someone was willing to write such a tool. Also python is much more contained and simpler to deal deal with than arbitrary dependencies.

Reply:
I was thinking about generating a manifest from a list of package names e.g. from the ubuntu or debian repositories, not from a .deb directly. This really shouldn't be any more difficult than the pip based case.. Or am I missing something?

Reply:
it wouldn't tell how to build them.